eslint.config.js

```diff
@@ -17,14 +17,31 @@ export default defineConfig([
   globalIgnores(['dist']),
   {
     extends: [
+      // https://github.com/eslint/eslint/blob/20a9e59438fde3642ab058cc55ee1b9fa02b6391/packages/js/src/configs/eslint-recommended.js
```

Comment:
Better to use `main` instead so that this comment doesn't get out of date over time?

```suggestion
      // https://github.com/eslint/eslint/blob/main/packages/js/src/configs/eslint-recommended.js
```